x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx

```diff
@@ -175,6 +180,14 @@ export class VectorLayer extends AbstractLayer implements IVectorLayer {
     });
   }
 
+  isEditable(): boolean {
+    const isEditable = this.getDataRequest(IS_EDITABLE_REQUEST_ID);
```

Comment:
"isEditable" sounds like the name for a boolean and it is confusing to see the variable used later  in "(isEditable.getData() as EditableData).isEditable". How about renaming "isEditable" to a name that describes what it is instead of name that describes how its used, like "dataRequest"

```
const dataRequest = this.getDataRequest(IS_EDITABLE_REQUEST_ID);
const data = dataRequest?.getData() as { isEditable: boolean } | undefined;
return data
    ? data.isEditable
    : false;
```